docs/docs/api/constants/app-killed-playback-behavior.md

```diff
@@ -0,0 +1,27 @@
+---
+sidebar_position: 6
+---
+
+# App Killed Playback Behavior (android-only)
+
+All iOS Category types are made available through the named export `AppKilledPlaybackBehavior`:
```

Comment:
Shouldn't this say Android Category?

Reply:
Dang... I shouldn't have merged so soon, sorry. I'll address all your feedback in a new PR.